oligo_designer_toolsuite/oligo_specificity_filter/_filter_blastn.py

```diff
@@ -103,6 +105,16 @@ def apply(self, database: dict, file_reference: str, n_jobs: int):
         return database
 
     def _create_index(self, file_reference: str, n_jobs: int):
+        """
+        Create a Blastn database index.
+
+        :param file_reference: Path to the reference file used for creating the index.
+        :type file_reference: str
+        :param n_jobs: Number of simultaneous parallel computations (currently unused)
+        :type n_jobs: int
```

Comment:
Good point, I kept it for consistency (because bowtie cas create an index using parallelism). I don't think it's possible to do it with Blast (I don't see any option for it here: https://github.com/biopython/biopython/blob/master/Bio/Blast/Applications.py#L1414)